Assets/UI Toolkit/Scripts/Components/Icon.cs

```diff
@@ -0,0 +1,58 @@
+﻿using Netherlands3D.UI;
+using Netherlands3D.UI.ExtensionMethods;
+using UnityEngine;
+using UnityEngine.UIElements;
+
+namespace Netherlands3D.UI.Components
+{
+    [UxmlElement]
+    public partial class Icon : VisualElement
+    {
+        public enum IconImage
```

Comment:
A supported list of images, these map onto the `.icon.image-X` class definitions in the Icon.uss